front/planning.php

```diff
@@ -36,6 +36,10 @@
    Session::checkRight("planning", READ);
 }
 
+if (isset($CFG_GLPI['planning_days'])) {
```

Comment:
I don't know why you do that and why the `getHiddenDays` function is needed.

If you need to have a difference between what you store and what you send to fullcalendar, you must:
- store as needed by fullcalendar (i think it's better, handling conversion leads to errors i think in this case)
- manage conversion on config loading (`Config::loadLegacyConfiguration`)

Reply:
so i store what fullcalendar except. But Config::loadLegacyConfiguration is not supposed to handle conversion for multiple dropdown in the config form. If i change behavior here, it works only for the planning himself but not anymore for general form (limit of days for planning)

Reply:
as i tried to is to stored hidden days in db. And manage conversion with function hiddenDays. It worked, but when you asked me to delete it. I'm facing misunderstanding about what you expect exactly about that, sorry if i didn't understand the whole issue